blog/2022-12-15-opencost-ui/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+slug: opencost-ui
+title: Using the OpenCost UI
+authors: [mattray]
+tags: [opencost]
+---
+![OpenCost UI](./img/ui-screenshot.png)
+
+The OpenCost UI is now available as a user-friendly interface for managing Kubernetes cloud costs. It allows users to easily monitor and visualize their cloud spend, making it easier to reduce Kubernetes costs while improving efficiency.
+
+To install the OpenCost UI you can use the container provided at (https://quay.io/repository/kubecost1/opencost-ui), which is now included in the standard [installation instructions](https://www.opencost.io/docs/install). The [opencost.yaml](https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/opencost.yaml) has been updated to deploy the container as part of the pod and listen on port 9090.
+
+You can upgrade or install fresh with
+```
+kubectl apply --namespace opencost -f https://raw.githubusercontent.com/opencost/opencost/develop/kubernetes/opencost.yaml
+```
+
+Expose the UI locally on port 9090
+```
+$ kubectl port-forward --namespace opencost service/opencost 9003 9090
+```
+and you can access it on [http://localhost:9090/]
```

Comment:
Meant to be a link? If not, should we remove brackets?

Reply:
Removed